runtimes/runtimes/standalone.ts

```diff
@@ -181,6 +182,61 @@ export const standalone = (props: RuntimeProps) => {
                         })
                     }),
                 getFileSize: path => stat(path),
+                getServerDataFolder: serverName => {
```

Comment:
It would be good to unit test this logic.

Reply:
Refactored this function to a separate file and added unit tests.